boards/xg23b-pk6068a/Makefile.features

```diff
@@ -0,0 +1,9 @@
+CPU = efm32
+CPU_FAM = efr32zg23
+CPU_MODEL = efr32zg23a020f512gm48
+
+# Put defined MCU peripherals here (in alphabetical order)
```

Comment:
What about the features that other SiliconLabs MCUs have?
`periph_adc`
`periph_rtc`
`periph_rtt`
`periph_uart_modecfg`

Reply:
Not implemented, yet. I'll add them on a follow-up PR. `periph_pwm` and `periph_dac` are missing, too ;-)

Reply:
Will you add them to the PR or as separate PRs? You should add entries for them in the task list in former case.

Reply:
Completely separate PR. This PR is already kinda huge.